srcpkgs/sview/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'sview'
+pkgname=sview
+version=20.08
+revision=1
+archs="i686 x86_64"
+wrksrc="sview-20_08"
+hostmakedepends="codeblocks ffmpeg libopenal libavcodec libavdevice libavformat libavutil libswscale libwebp libconfig++ libconfig libXpm pkg-config gtk+ libXrandr"
+makedepends="ffmpeg-devel libopenal-devel libwebp-devel libconfig++-devel libconfig-devel libXpm-devel gtk+-devel libXrandr-devel"
+depends="libopenal libavcodec libavdevice libavformat libavutil libswscale libwebp libconfig++ libconfig libXpm"
```

Comment:
Pretty much all `depends` here should be automatically detected by the build system, you don't need to list them.

Reply:
They are: https://travis-ci.org/github/void-linux/void-packages/jobs/731797076#L1117